status.sh

```diff
@@ -9,6 +9,7 @@ set -o pipefail
 # NOMAD_ADDR=""
 # NOMAD_NAMESPACE=""
 # PARAMETERIZED_JOB="(true|false)"
+# ALLOW_FAILURE="(true|false)"
```

Comment:
Same, here, the ENV var name is not that good here.

Reply:
renamed to ALLOW_STDERR